modules/form-builder/main/index.coffee

```diff
@@ -21,6 +20,10 @@ class Form extends hx.EventEmitter
     selection = entry.append(nodeType).attr("id",id)
     extras = f.call(selection) or {}
 
+    # Define the default function for enabling/disabling a form property
+    extras.disable ?= (sel, disable) ->
```

Comment:
Is this what is exposed in the public api? If so, can it be made a getter setter?